### ISSUE TITLE
metrics api call out of memory

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/syrup-queue#53]
`GET /queue/metrics?dateFrom=2017-01-01&dateTo=2017-02-15&period=month HTTP/1.1` => 500 App Error

```
PHP Fatal error:  Allowed memory size of 268435456 bytes exhausted (tried to allocate 1048577 bytes) in /www/syrup-router/components/queue/releases/20170124200942/vendor/guzzlehttp/psr7/src/Stream.php on line 211
```
exceptionId: queue-a31c9ed9683aa34e46d981a0b495b9a6

**Comment by ujovlado** [comment]
Ciastocne vyriesene v ramci #57

**Comment by ujovlado** [comment]
Fixed in #57
